assets/js/translator.js

```diff
@@ -58,6 +83,15 @@ if(modeEnabled('translation')) {
             persistChoices('translator', true);
         });
 
+        $('input.chainedTranslation').change(function () {
```

Comment:
I'm using the class for hiding/showing the checkbox and the text, while I'm adding an id to get the state of the checkbox.